### PR TITLE
Backend rmsd

### PR DESCRIPTION
**Pull request by jmiller656**
Add RMSD calculation to backend response for alignment

### CODE REVIEW

**Comment by blackpan2** [comment]
Can we mark this as blocked until https://github.com/moltimate/moltimate-backend/pull/74 is merged since this appears to contain a subset of changes? @jmiller656

**Comment by jmiller656** [comment]
Yeet. Probably good to verify that the numbers look right with sponsors first too, actually. I think they look good because <1 but they might need to be smaller

File: src/main/java/org/moltimate/moltimatebackend/util/AlignmentUtils.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package org.moltimate.moltimatebackend.util;
 
 import org.biojava.nbio.structure.Group;
+import org.biojava.nbio.structure.geometry.SuperPositionSVD;
```
Comment (steplica) [inline src/main/java/org/moltimate/moltimatebackend/util/AlignmentUtils.java]: Unused import?

Comment (jmiller656) [reply src/main/java/org/moltimate/moltimatebackend/util/AlignmentUtils.java]: Oops, fixing

**Pull request merged**